src/storage/src/hummock/event_handler/mod.rs

```diff
@@ -64,4 +67,75 @@ pub enum HummockEvent {
     /// Flush all previous event. When all previous events has been consumed, the event handler
     /// will notify
     FlushEvent(oneshot::Sender<()>),
+    RegisterHummockInstance {
+        table_id: TableId,
+        instance_id: u64,
+        hummock_version_reader: HummockVersionReader,
```

Comment:
If the reader is the same for all local instances, we may store the reader in the event handler and clone when we handle the request.

Reply:
I am thinking, the event handler only needs the read version, and other necessary fields to initialize the local hummock is in the scope of `new_local`. Therefore, we may make this event `RegisterNewReadVersion`, which requests for a new `ReadVersion` instead of a new local instance. Besides, the instance id can be returned from the event handler instead of passed from outside.